src/roadrunner-symfony-nyholm/composer.json

```
@@ -12,14 +12,16 @@
     "require": {
         "php": ">=8.0.5",
         "nyholm/psr7": "^1.4",
-        "spiral/roadrunner": "^2.0",
+        "spiral/roadrunner-cli": "^2.6",
+        "spiral/roadrunner-http": "^2.2 || ^3.5.1",
         "symfony/dependency-injection": "^5.4 || ^6.0 || ^7.0",
         "symfony/http-kernel": "^5.4 || ^6.0 || ^7.0",
         "symfony/psr-http-message-bridge": "^2.1 || ^6.4 || ^7.0",
         "symfony/runtime": "^5.4 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5"
+        "phpunit/phpunit": "^9.5",
+        "symfony/console": "^6.4 || ^7.0"
```

Is 5.4 not possible?

```suggestion
        "symfony/console": "^5.4 || ^6.4 || ^7.0"
```

---

@alexander-schranz Locally, with `symfony/console` 5.4, I get the error **ini_set(): assert.warning INI setting is deprecated** on PHP 8.3. This is a **dev** dependency, so I didn’t add this version. Should I add it and see if it breaks the CI?

---

We should add it to avoid that accidently lowest tests are not longer running on Symfony 5

---

Added ^5.4